src/command_modules/azure-cli-netappfiles/azure/cli/command_modules/netappfiles/_exception_handler.py

```diff
@@ -0,0 +1,19 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+from msrest.exceptions import ValidationError  # pylint: disable=import-error
+from msrestazure.azure_exceptions import CloudError
+
+
+def netappfiles_exception_handler(ex):
+    if isinstance(ex, CloudError) or isinstance(ex, ValidationError) or isinstance(ex, ValueError):
```

Comment:
This seems very generic. The default CLI exception handler should be sufficient.

Reply:
> This seems very generic. The default CLI exception handler should be sufficient.

Ok. What is the default CLI exception? Should I not capture the CloudError? The ValidationError and ValueError were added deliberately because otherwise certain invalid commands would result in a trace log which didn't seem to be what a CLI user would want to see. Please advise further.